Makefile

```diff
@@ -12,7 +12,7 @@ export PKG_CONFIG:=$(shell pwd)/pkg-config.sh
 
 export GOOS=$(shell go env GOOS)
 export GO_BUILD=env GO111MODULE=on go build $(GO_ARGS)
-export GO_TEST=env GO111MODULE=on go test $(GO_ARGS)
```

Comment:
I believe this is defined in `.circleci/config.yml`. We inject `assert` through an argument.

Reply:
Seems like if we go that route, we won't get the nice feedback when testing locally. I'm fine with reverting it, but it seems a bit odd :woozy_face: